jenkins/gradle-check-assemble.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```
#!/usr/bin/env searches PATH for bash, and bash is not always in /bin, particularly on non-Linux systems. For example, on my OpenBSD system, it's in /usr/local/bin, since it was installed as an optional package.

https://stackoverflow.com/questions/21612980/why-is-usr-bin-env-bash-superior-to-bin-bash/21613044#21613044